gradle-plugins/model-compiler/src/test/java/io/spine/tools/compiler/annotation/check/Annotations.java

```diff
@@ -18,16 +18,35 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+package io.spine.tools.compiler.annotation.check;
+
+import io.spine.annotation.SPI;
+import org.jboss.forge.roaster.model.source.AnnotationSource;
+import org.jboss.forge.roaster.model.source.AnnotationTargetSource;
+
+import javax.annotation.Nullable;
+import java.lang.annotation.Annotation;
+
 /**
- * Provides a Gradle plugin, which adds annotations for generated Java files
- * basing on options defined in {@code .proto} files.
- *
- * <p>Contains classes and interfaces for applying annotations on Java files,
- * which are generated from Protobuf definitions.
- *
- * @see io.spine.gradle.compiler.annotation.ProtoAnnotatorPlugin ProtoAnnotatorPlugin
+ * @author Dmytro Grankin
+ * @author Alexander Yevsyukov
  */
-@ParametersAreNonnullByDefault
-package io.spine.gradle.compiler.annotation;
+class Annotations {
```

Comment:
This class seems to miss the class-level Javadocs.

Reply:
@alexander-yevsyukov please disregard this comment. I confused it with the main-source class, while it is a test-source one.

Reply:
I added the doc anyway. It won't hurt :)